services/secondary-storage/controller/src/main/java/org/apache/cloudstack/secondarystorage/SecondaryStorageManagerImpl.java

```diff
@@ -401,6 +401,9 @@ private List<String> getAllowedInternalSiteCidrs() {
         String[] cidrs = _allowedInternalSites.split(",");
         for (String cidr : cidrs) {
             if (NetUtils.isValidIp4Cidr(cidr) || NetUtils.isValidIp4(cidr) || !cidr.startsWith("0.0.0.0")) {
+                if (NetUtils.getCleanIp4Cidr(cidr).equals(cidr)) {
+                    s_logger.warn(String.format("Invalid CIDR %s in secstorage.allowed.internal.sites", cidr));
```

Comment:
```suggestion
                    s_logger.warn(String.format("Invalid CIDR %s in the config %s", cidr, SecStorageAllowedInternalDownloadSites.key()));
```

try import SecStorageAllowedInternalDownloadSites if possible, otherwise keep 'secstorage.allowed.internal.sites'.